src/frontend/src/components/icons.ts

```diff
@@ -165,3 +165,32 @@ export const warningIcon = html`
     />
   </svg>
 `;
+
+export const securityKeyIcon = html`
+  <svg
+    width="53"
+    height="52"
+    viewBox="0 0 53 52"
+    fill="none"
+    xmlns="http://www.w3.org/2000/svg"
+  >
+    <path
+      d="M39.0938 21.9375V7.3125C39.0938 6.41367 38.3676 5.6875 37.4688 5.6875H15.5312C14.6324 5.6875 13.9062 6.41367 13.9062 7.3125V21.9375C10.5445 21.9375 7.8125 24.5832 7.8125 27.8281V45.9062C7.8125 46.1297 7.99531 46.3125 8.21875 46.3125H11.0625C11.2859 46.3125 11.4688 46.1297 11.4688 45.9062V27.8281C11.4688 26.5941 12.5656 25.5938 13.9113 25.5938H39.0887C40.4344 25.5938 41.5312 26.5941 41.5312 27.8281V45.9062C41.5312 46.1297 41.7141 46.3125 41.9375 46.3125H44.7812C45.0047 46.3125 45.1875 46.1297 45.1875 45.9062V27.8281C45.1875 24.5832 42.4555 21.9375 39.0938 21.9375V21.9375ZM17.5625 21.9375V9.34375H35.4375V21.9375H17.5625ZM23.6562 12.5938H21.2188C20.9953 12.5938 20.8125 12.7766 20.8125 13V15.4375C20.8125 15.6609 20.9953 15.8438 21.2188 15.8438H23.6562C23.8797 15.8438 24.0625 15.6609 24.0625 15.4375V13C24.0625 12.7766 23.8797 12.5938 23.6562 12.5938ZM31.7812 12.5938H29.3438C29.1203 12.5938 28.9375 12.7766 28.9375 13V15.4375C28.9375 15.6609 29.1203 15.8438 29.3438 15.8438H31.7812C32.0047 15.8438 32.1875 15.6609 32.1875 15.4375V13C32.1875 12.7766 32.0047 12.5938 31.7812 12.5938Z"
+      fill="#292A2E"
+    />
+  </svg>
+`;
+export const seedPhraseIcon = html`
+  <svg
+    width="53"
+    height="52"
+    viewBox="0 0 53 52"
+    fill="none"
+    xmlns="http://www.w3.org/2000/svg"
+  >
+    <path
+      d="M42.75 23.5625H39.2969V12.1875C39.2969 8.59727 36.3871 5.6875 32.7969 5.6875H20.2031C16.6129 5.6875 13.7031 8.59727 13.7031 12.1875V23.5625H10.25C9.35117 23.5625 8.625 24.2887 8.625 25.1875V44.6875C8.625 45.5863 9.35117 46.3125 10.25 46.3125H42.75C43.6488 46.3125 44.375 45.5863 44.375 44.6875V25.1875C44.375 24.2887 43.6488 23.5625 42.75 23.5625ZM17.3594 12.1875C17.3594 10.6184 18.634 9.34375 20.2031 9.34375H32.7969C34.366 9.34375 35.6406 10.6184 35.6406 12.1875V23.5625H17.3594V12.1875ZM40.7188 42.6562H12.2812V27.2188H40.7188V42.6562ZM25.0781 35.5977V38.2891C25.0781 38.5125 25.2609 38.6953 25.4844 38.6953H27.5156C27.7391 38.6953 27.9219 38.5125 27.9219 38.2891V35.5977C28.3411 35.2967 28.6539 34.8705 28.8155 34.3804C28.977 33.8903 28.9789 33.3616 28.8208 32.8704C28.6627 32.3791 28.3529 31.9508 27.9358 31.6468C27.5188 31.3429 27.016 31.1792 26.5 31.1792C25.984 31.1792 25.4812 31.3429 25.0642 31.6468C24.6471 31.9508 24.3373 32.3791 24.1792 32.8704C24.0211 33.3616 24.023 33.8903 24.1845 34.3804C24.3461 34.8705 24.6589 35.2967 25.0781 35.5977Z"
+      fill="#292A2E"
+    />
+  </svg>
```

Comment:
Tangential, would it be worth putting these actual svgs into the repo embed them from there (using `declare module` or something with webpack)?

Reply:
I'm not sure if that works... type wise these are `TemplateResult`s as opposed to strings, so we don't end up reparsing them with the template library over and over.